python/tests/parsers/test_openai_util.py

```diff
@@ -46,11 +46,11 @@ def test_refine_chat_completion_params():
     assert "random_attribute" not in refined_params
     assert refined_params["n"] == "3"
 
-@pytest.mark.xfail
+
 @pytest.mark.asyncio
 async def test_get_output_text(set_temporary_env_vars):
     with patch.object(
-        openai.chat.completions,
+        openai.resources.chat.Completions,
```

Comment:
how did you figure out that you need to use resources to fix this?

Reply:
I just read through the comments https://github.com/openai/openai-python/issues/715#issuecomment-1809203346 and saw they were patching it that way. Tried it out and it worked :)